cli/tox.ini

```diff
@@ -211,7 +211,7 @@ changedir =
     ../cloudformation
 deps = cfn-lint
 commands =
-    cfn-lint --info *.cfn.json --ignore-templates fsx-substack.cfn.json
+    cfn-lint --info *.cfn.json --ignore-templates fsx-substack.cfn.json --ignore-checks E3030
```

Comment:
if `E3030` cannot be fixed can we add a comment explaining why?

Reply:
Done. Linked to [the issue for it](https://github.com/aws-cloudformation/cfn-python-lint/issues/1588).

Reply:
Thanks for the report! [Rules can also be ignored at a resource level](https://github.com/aws-cloudformation/cfn-python-lint/#resource-based-metadata) if you just want to ignore it for one resource

Reply:
I wasn't aware of that. Thanks for the tip.